skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -1,136 +1,169 @@
 			return <img src={require('../components/navbar-icons/create-red.png')}
-						className="create-icon-grey" alt='Create tab Selected'></img>
+				className="create-icon-grey" alt='Create tab Selected'></img>
 		}
 		else {
 			return <img src={require('../components/navbar-icons/create-grey.png')}
-						className="create-icon-grey" alt='Create tab not Selected'></img>
+				className="create-icon-grey" alt='Create tab not Selected'></img>
 		}
 	}
 
-	isScoreCapture(){
+	isScoreCapture() {
 		if (window.location.pathname.endsWith("/scorecapture")) {
 			return <img src={require('../components/navbar-icons/add-score-red.png')}
-						className="add-score-icon-grey" alt='ScoreCapture tab Selected'></img>
+				className="add-score-icon-grey" alt='ScoreCapture tab Selected'></img>
 		}
 		else {
 			return <img src={require('../components/navbar-icons/add-score-grey.png')}
-						className="add-score-icon-grey" alt='ScoreCapture tab not Selected'></img>
+				className="add-score-icon-grey" alt='ScoreCapture tab not Selected'></img>
 		}
 	}
 
-	isProfile(){
+	isProfile() {
 		if (window.location.pathname.endsWith("/profile")) {
 			return <img src={require('../components/navbar-icons/profile-red.png')}
-						className="profile-icon-grey" alt='Profile tab Selected'></img>
+				className="profile-icon-grey" alt='Profile tab Selected'></img>
 		}
 		else {
 			return <img src={require('../components/navbar-icons/profile-grey.png')}
-						className="profile-icon-grey" alt='Profile tab not Selected'></img>
+				className="profile-icon-grey" alt='Profile tab not Selected'></img>
 		}
 	}
 
-	isDocuments(){
+	isDocuments() {
 		if (window.location.pathname.endsWith("/documents")) {
 			return <img src={require('../components/navbar-icons/docs-red.png')}
-						className="docs-icon-grey{" alt='Document tab Selected'></img>
+				className="docs-icon-grey" alt='Document tab Selected'></img>
 		}
 		else {
 			return <img src={require('../components/navbar-icons/docs-grey.png')}
-						className="docs-icon-grey" alt='Document tab not Selected'></img>
+				className="docs-icon-grey" alt='Document tab not Selected'></img>
 		}
 	}
 
-	isNotifications(){
+	isNotifications() {
 		if (window.location.pathname.endsWith("/notifications")) {
 			return <img src={require('../components/navbar-icons/notifications-red.png')}
-						className="notifications-icon-grey" alt='Notification tab Selected'></img>
+				className="notifications-icon-grey" alt='Notification tab Selected'></img>
 		}
 		else {
 			return <img src={require('../components/navbar-icons/notifications-grey.png')}
-						className="notifications-icon-grey" alt='Notification tab not Selected'></img>
+				className="notifications-icon-grey" alt='Notification tab not Selected'></img>
 		}
 	}
 
+	isMore() {
+		return <img src={require('../components/navbar-icons/more-grey.png')}
+			className="more-icon-grey" alt='More icon to expand tray'></img>
+	}
+
+	GoToLeaderboard (){
```

Comment:
Make a generic function for this. 
add a todo. 
Dm me I will help you set it up

Reply:
will do